src/Runner.cs

```diff
@@ -113,10 +113,18 @@ public void Processor(object data)
         {
             string path = (string)data;
 
+            string fileName = "*.*";
+
+            if (File.Exists(path))
+            {
+                fileName = Path.GetFileName(path);
+                path = path.Substring(0, path.Length - fileName.Length);
```

Comment:
I think `Path.GetDirectoryName(path)` would be clearer here as it reads out as *what* is being done instead of *how* it is implemented.